backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -242,7 +242,7 @@ def __processBootsTest(self, currentRowData):
             self.bootFailReasons[testError] = self.bootFailReasons.get(testError, 0) + 1
 
         if testEnvironmentCompatible is not None:
-            self.bootEnvironmentCompatible[testEnvironmentCompatible] += 1
+            self.bootEnvironmentCompatible[testEnvironmentCompatible][testStatus] += 1
```

Comment:
Some test status are coming as null nowadays, I whink we should acknowledge for them and count them

Reply:
I added this logic, calling null testStatus as `none` or `NONE` to avoid errors with `null` JavaScript